ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.6"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+canonical_go_repository: github.com/openshift/route-monitor-operator
+promotion:
+  name: "4.6"
+  namespace: ocp
```

Comment:
Remove these

Reply:
Don't these mean that we follow a normal release cycle like other operators?